src/axom/spin/policy/LinearBVH.hpp

```diff
@@ -36,6 +36,24 @@ namespace policy
 {
 namespace lbvh = internal::linear_bvh;
 
+/*
+ * \brief Interface for traversing a BVH tree.
+ *
+ * \brief Traverse a tree to perform user-specified actions at the
+ * leaves, while limiting the search to branches satisfying a
+ * user-provided predicate.
+ *
+ * To a initiate traversals, use traverse_tree.  It requires
```

Comment:
Typo and suggested highlight:
```suggestion
 * To initiate traversals, use \a traverse_tree.  It requires
```

Reply:
done.  thanks.